### ISSUE TITLE
Rule Change: (sort-imports)

### ISSUE THREAD

**Issue by sarkiroka** [eslint/eslint#15103]
### What rule do you want to change?

sort-imports

### What change to do you want to make?

Generate more warnings

### How do you think the change should be implemented?

A new option

### Example code

```js
When I sort the imports, I have to decide the type of imports: group(multiple), individual(single), all or none. I'm fine with the last two, but the order of the first two is not relevant for me, so I don't want to group the abc blocks separately in case they start with {.
For example this should be acceptable:

import a from './a.js';
import {b} from './b.js';
import c from './c.js';
import {d} from './d.js'
```
However, currently the memberSyntaxSortOrder option requires 4 values, two of which control just that.
I would like to ask that if this is possible, it is possible to specify the order with the following syntax:
```
memberSyntaxSortOrder: ["none", "all", "others"]
```
I don't insist on this keyword, the main thing is to be able to create a condition in the final result where multiple/single condition doesn't matter.
Currently I could achieve this with a hack like this, but it breaks the cli tool unfortunately:
```
memberSyntaxSortOrder: ["undefined", "all"]
```
```


### What does the rule currently do for this code?

Currently must decide which is first: groups or individual imports
```
import {b} from './b.js';
import {d} from './d.js'
import a from './a.js';
import c from './c.js';
```
or
```
import a from './a.js';
import c from './c.js';
import {b} from './b.js';
import {d} from './d.js'

```

### What will the rule do after it's changed?

After that I can import in abc order without single or multiple decide
```
import a from './a.js';
import {b} from './b.js';
import c from './c.js';
import {d} from './d.js'

```

### Participation

- [ ] I am willing to submit a pull request to implement this change.

### Additional comments

It is a great rule by the way, thanks for it

**Comment by mdjermanovic** [comment]
Hi @sarkiroka, thanks for the issue!

Per our rules policies, [stylistic rules are frozen](https://github.com/eslint/eslint#stylistic-rule-updates) for the reasons described in [this blog post](https://eslint.org/blog/2020/05/changes-to-rules-policies), so we won't be adding any new options to `sort-imports`, or changing the way how the current options work. You can always make a custom rule based on the core rule, and modify it to your preferences.

**Comment by sarkiroka** [comment]
Thanks your answer. I missed this frozen rule. I'm a bit sad now. Can you advise me what to do in this situation?
